src/main/java/com/faforever/api/config/MvcConfig.java

```diff
@@ -31,28 +26,12 @@ public void addResourceHandlers(ResourceHandlerRegistry registry) {
       .addResourceLocations("classpath:/META-INF/resources/");
   }
 
-  @Override
-  public void configurePathMatch(PathMatchConfigurer configurer) {
-    configurer.setUseRegisteredSuffixPatternMatch(true);
-  }
-
-  @Override
-  public void configureContentNegotiation(final ContentNegotiationConfigurer configurer) {
-    // Turn off suffix-based content negotiation
-    configurer.favorPathExtension(false);
-  }
-
   @Override
   public void addCorsMappings(CorsRegistry registry) {
     registry.addMapping("/**")
       .allowedMethods("*");
   }
 
-  @Override
-  public void extendMessageConverters(List<HttpMessageConverter<?>> converters) {
```

Comment:
I guess this was some workaround. Now it caused UploadMetadata to not be parsed at all